server/neptune/workflows/internal/activities/terraform.go

```diff
@@ -30,15 +30,22 @@ type TerraformClient interface {
 	RunCommand(ctx context.Context, jobID string, path string, subcommand *terraform.SubCommand, customEnvVars map[string]string, v *version.Version) <-chan terraform.Line
 }
 
+type outputReader interface {
```

Comment:
I would probably call this the opposite tbh, `outputWriter` since even though it's reading it, it's not giving you back that info, it's writing it somewhere.